karspexet/templates/base.html

```diff
@@ -0,0 +1,91 @@
+{% load menu_tags cms_tags sekizai_tags staticfiles%}
+<html>
+    <head>
+        <title>{% page_attribute "page_title" %}</title>
+        <link rel="shortcut icon" href="{% static 'favicon.ico' %}">
+        {% render_block "css" %}
+    </head>
+    <body>
+        {% cms_toolbar %}
+        <div id="navbar">
+            <div class="horizontal titleContainer">
+              <h1><span class="g">KÅR</span>S<span class="g">P</span>EX<span class="g">ET</span></h1>
+            </div>
+            
+            <div class="horizontal menuContainer">
+              <ul class="menu">
+                {% show_menu 0 100 0 1 %}
+              </ul>
+            </div>
+        </div>
+        {% addtoblock "css" %}
+        <link href="{% static 'fonts.css' %}" rel="stylesheet" type="text/css"/>
+        <link href="{% static 'menu.css' %}" rel="stylesheet" type="text/css"/>
+        {% endaddtoblock %}
+        {% addtoblock "js" %}
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```

Comment:
No worries, it's not obvious and until that incident people didn't really consider it dangerous.